### DISCUSSION TITLE
How to include the activity of a source?

### DISCUSSION THREAD

**Discussion by merleluise** [clrp-code/egs_brachy#58] [Q&A] [answered]
Dear all, 
I am absolutely new in MC simulations/EGSnrc and want to simulate an interstitial brachytherapy with egs_brachy, to compare the dose results with our experimental results. 
Now my problem is, that I can not find if or how I can insert the activity of the sources in the simulation and then consider the emission time or the dwell positions/times to simulate/calculate the absorbed dose in my phantom. 
Any help or references to literature would be greatly appreciated!

Thanks!

**Comment by randlet** [comment, marked as answer]
I'm pretty sure this is discussed in the egs_brachy user manual (distributed with egs_brachy or available here: https://clrp-code.github.io/egs_brachy/pdf/egs_brachy_user_manual.pdf).   Searching for "dwell" should find the right section. THere's also an example input file ex_PeppaBreastHDR192Ir.egsinp (also mentioned in the manual) that uses different dwell times.

**Comment by mchamberland** [comment]
@merleluise  Just want to add a few things in addition to what Randy wrote.

Say you have two identical sources: one with activity 1 C, the other, 10 Ci. Then, you expect the dose distribution around each source to be identical, except for a scaling factor equal to 10 (i.e., the ratio of the source activities). Similarly, if you have two identical HDR sources, one that only dwells for 1 second, the other for 10 seconds, then again, the dose distributions between the two only differ by a scaling factor of 10.

All this to say that: in egs_brachy, activity and time can be controlled via the `dose scaling factor` key of the `scoring options` input block. All this does is scale your resulting dose distribution by a scaling factor. So, it is your job to determine what exactly is that scaling factor.

You mention dwell positions and times. I assume you are doing an HDR simulation? In that case, you need to use the `superposition `run mode and the `egs_aswitchedenvelope `geometry for the dwell positions and the `source weights` input from the source definition for your dwell times. Again, look at what Randy pointed to: the ex_PeppaBreastHDR192Ir.egsinp example input file.

And if you really need to do absolute dosimetry, then you will also need your source air-kerma strength as well as the air-kerma strength per history factor, which is calculated from a simulation under calibration conditions. If you're using a real commercially available HDR source, then there's a good chance that the CLRP group has already calculated the air-kerma strength per history factor and I can provide it to you.

Please don't hesitate to reach out if you have more questions.

**Reply by merleluise** [reply]
Thank you both for the quick reply! 
I struggeled a bit because the example doesn't work for me so far, but your answers were still very helpful and I will try my hand at it, thank you very much!
@mchamberland we use a microSelectron v3 HDR192Ir source

**Reply by mchamberland** [reply]
@merleluise Is v3 referring to the afterloader or the actual source? I am not familiar with a v3 of the source. I only know of the v2 (and the revision to the v2, which is sold as v2 still).

See [here](https://www.estro.org/About/ESTRO-Organisation-Structure/Committees/GEC-ESTRO-Committee/GEC-ESTRO-BRAPHYQS/Ir-192-HDR), [here](https://physics.carleton.ca/clrp/nucletron-microselectron-v2r-mhdr-v2r), and [here](https://aapm.onlinelibrary.wiley.com/doi/10.1002/mp.16176).

**Reply by merleluise** [reply]
@mchamberland yeah, you are totally right, I looked it up and our source is a **v2** and only our system is v3!

**Reply by mchamberland** [reply]
@merleluise I'll try to get you the air-kerma strength per history factor for this source early next week.

Please don't hesitate to reach out if you're still struggling or have any more questions. There is some learning curve with EGSnrc and Monte Carlo in general.

**Reply by mchamberland** [reply]
@merleluise Have not forgotten about this. Busy week in the clinic!

**Reply by mchamberland** [reply]
@merleluise The air-kerma strength per history factor for the microSelectron v2 (revision) is 1.1633e-13 Gy*cm^2/history.

This is based on the data published in this paper: [https://aapm.onlinelibrary.wiley.com/doi/10.1002/mp.16176](https://aapm.onlinelibrary.wiley.com/doi/10.1002/mp.16176)   (but this actual value is not reported in the publication).

**Reply by merleluise** [reply]
@mchamberland thank you very much, that helps me a lot! Here it was also stressful, so thank you so much that it worked out!
